src/ansiblelint/rules/yaml.md

```diff
@@ -72,6 +73,11 @@ for it does check these. If for some reason, you do not want to follow our
 defaults, you can create a `.yamllint` file in your project and this will take
 precedence over our defaults.
 
+## Additional Information
```

Comment:
should we mention the title as `Additional Information For Multiline Strings`.
That way user would be able to directly reference it instead of searching for multiline related info

Reply:
I made the title as `Additional Information` so that if in future we need to add in other yaml rule specific info, it can directly go inside this title. But for now, we can name it as `Additional Information For Multiline Strings`.

Reply:
@ajinkyau Done with the changes!